ui/src/components/forms/Form.js

```diff
@@ -202,6 +203,15 @@ const Form = (props) => {
             </Select>
           );
           break;
+        case 'ministryTeam':
```

Comment:
idk if it makes sense for a form to ask for ministry team? thought we agreed on that this is more manageable for admins to just go in and change by themselves instead of allow users to update their own ministry teams